packages/bierzo-wallet/src/components/Wallet/CurrencyToSend/index.tsx

```diff
@@ -0,0 +1,65 @@
+import Avatar from '@material-ui/core/Avatar';
+import Paper from '@material-ui/core/Paper';
+import Select from '@material-ui/core/Select';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/styles';
+import React from 'react';
+
+const StyledPaper = withStyles({
```

Comment:
```withStyles``` usage AFAIK, is discouraged. If you want to use styled 💅 components, you should go with:
https://next.material-ui.com/css-in-js/basics/#styled-components-api

But I recommend the hook way, which is the one we are using in the extension:
https://next.material-ui.com/css-in-js/basics/#hook-api

Reply:
I tried both approaches but I couldn't find a way to override the inner classes. That's why I chose the "Shorthand" approach that can be found here:
[https://material-ui.com/customization/overrides/](https://material-ui.com/customization/overrides/)

Reply:
Fixed by [3daf67f](https://github.com/iov-one/ponferrada/pull/79/commits/3daf67fea9627053c6c04109b97b5e0b2e8a2b46).